CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ### main
 - Added roundtrip of unrecognized JSON properties. [#1394](https://github.com/mapbox/mapbox-java/pull/1394)
 - Added API to access unrecognized JSON properties. See `#getUnrecognizedProperty` and `#getUnrecognizedPropertiesNames` in each Directions API model object.  [#1416](https://github.com/mapbox/mapbox-java/pull/1416)
+- Added `RouteOptions.Builder#unrecognizedProperties` to set custom request parameters for `RouteOptions`. [#1420](https://github.com/mapbox/mapbox-java/pull/1420)
```

Comment:
Same as above, please not that this works only for `toUrl`.

Reply:
done